index.html

```diff
@@ -118,7 +118,7 @@ <h2>Tonic</h2>
                             <li class="tag">javascript</button>
                         </ul>
                         <div class="ref">
-                        <button>See project</button>
+                        <button>Seee project</button>
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:
- [ ] Kindly also make changes to your `Get my resume` button.

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2` and on hover has the `background: #6070FF`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```


![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```